apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/Trace/TraceTest.cs

```diff
@@ -138,7 +139,8 @@ private void BlockUntilClockTick()
         public async Task Trace_Simple_Buffer()
         {
             string rootSpanName = CreateRootSpanName(nameof(Trace_Simple_Buffer));
-            BufferingTraceConsumer consumer = BufferingTraceConsumer.Create(CreateGrpcTraceConsumer());
+            SizedBufferingConsumer<TraceProto> consumer = SizedBufferingConsumer<TraceProto>.Create(
```

Comment:
For tests, I'd start using `var` for this sort of thing. You might also consider having extension methods on `IConsumer<T>`, so you could have

```csharp
var consumer = CreateGrpcTraceConsumer.BufferedBySize(TraceSizer.Instance, BufferOptions.DefaultBufferSize);
```

... which I assume with optional parameters could then be equivalent to


```csharp
var consumer = CreateGrpcTraceConsumer.BufferedBySize();
```